stripper/ze_SUMAI_facility_v1a.cfg

```diff
@@ -10,4 +10,35 @@ modify:
 	{
 		"spawnflags" "2"
 	}
-}
\ No newline at end of file
+}
+
+;Remove HTML formatting broken/non-functional after the Shattered Web Update
+;############################## THIS FIX WILL NOT WORK WITHOUT HAVING ####################################
+;########################### csgo/scripts/vscripts/darnias/bosshp.nut ####################################
+;###################################### IN THE SERVER FILES ##############################################
+;######## https://github.com/gflclan-cs-go-ze/ZE-Configs/blob/master/vscripts/bosshp.nut #################
```

Comment:
Should be bosshp_patched.nut instead of bosshp.nut. There's also no mention of fanshp_patched.nut.